app/assets/javascripts/discourse/app/lib/plugin-api.js

```diff
@@ -136,7 +136,7 @@ import { modifySelectKit } from "select-kit/mixins/plugin-api";
 // docs/CHANGELOG-JAVASCRIPT-PLUGIN-API.md whenever you change the version
 // using the format described at https://keepachangelog.com/en/1.0.0/.
 
-export const PLUGIN_API_VERSION = "1.15.0";
+export const PLUGIN_API_VERSION = "1.16.0";
```

Comment:
Does `reciprocallyRankedList` need to be exposed in the plugin API? It looks like it isn't necessary, in which case we don't need to bump the version.

Reply:
Ah yes good catch, thanks!